brain_observatory_analysis/dev/data_selection_tools.py

```diff
@@ -0,0 +1,139 @@
+import os
+import json
+# import warnings
+import numpy as np
+import pandas as pd
+from pathlib import Path
+
+# import logging
+# logger = logging.getLogger(__name__)
+
+
+def add_layer_column(df):
+    """
+    Adds a column called 'layer' that is based on the 'imaging_depth' for each experiment.
+    if imaging_depth is <250um, layer is 'upper, if >250um, layer is 'lower'
+    :param df:
+    :return:
+    """
+    df.loc[:, 'layer'] = None
+
+    indices = df[(df.depth < 250)].index.values
+    df.loc[indices, 'layer'] = 'upper'
+
+    indices = df[(df.depth > 250)].index.values
+    df.loc[indices, 'layer'] = 'lower'
+
+    return df
+
+def add_area_layer_column(df):
```

Comment:
If the above function is getting removed, then this one can be removed or fixed.